src/util.h

```diff
@@ -6,7 +6,7 @@
 
 #include <k.h>
 
-#if (KXVER==3) || (KXVER == 4)
+#if (KXVER==3) || (KXVER==4)
```

Comment:
might ``#if KXVER>2`` be more reasonable?

Reply:
changed, I don't know how likely the k.h names are to change in the future. 

Btw, I just finished qtips and it was really useful, thanks. It's just the sort of resource I wish I would have had, when I did kdb full time 5 years ago.